features/dataset-view.feature

```diff
@@ -185,24 +185,33 @@ Feature: a user visit the dataset page
 		Given I am not logged in to Gigadb web site
 		When I go to "/dataset/101001"
 		Then I should see "Files" tab with table
-		| File name              							| Sample ID  	| Data Type       	| File Format 	| Size  		| Release date| link |
-		| Anas_platyrhynchos.cds 							| Pekin duck 	| Coding sequence  	| FASTA 	   	| 21.50 MiB     | 2015-08-03  | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/Anas_platyrhynchos.cds |
-	 	| Anas_platyrhynchos.gff 							| Pekin duck 	| Annotation 		| GFF        	| 10.10 MiB 	| 2015-08-03  | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/Anas_platyrhynchos.gff |
-		| Anas_platyrhynchos.pep 							| Pekin duck 	| Protein sequence 	| FASTA      	| 7.80 MiB  	| 2015-08-03  | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/Anas_platyrhynchos.pep |
-		| Anas_platyrhynchos_domestica.RepeatMasker.out.gz 	| Pekin duck 	| Other 			| UNKNOWN    	| 7.79 MiB  	| 2015-03-23  | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/Anas_platyrhynchos_domestica.RepeatMasker.out.gz |
-		| duck.scafSeq.gapFilled.noMito 					| Pekin duck 	| Sequence assembly	| FASTA 		| 1.03 GiB 		| 2013-01-23  | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/duck.scafSeq.gapFilled.noMito |
-		| pre_03AUG2015_update 								|				| Directory 		| UNKNOWN 		| 50.00 MiB 	| 2015-08-03  | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/pre_03AUG2015_update |
-		| readme.txt 										|				| Readme 			| TEXT 			| 337 B 		| 2013-01-23  | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/readme.txt |
+		| File Name              							| Description  	                                                                    | Data Type       	| Size  		| File Attributes | link |
```

Comment:
Try changing `File Name` to `File name` and re-run the this test to see if it passes.

Reply:
@pli888 The behat tests for all scenario have been passed after changing to `File name`. At first, I changed it to `File Name` because the `Name` appears in the files table of local gigadb web page.

Reply:
Hi @kencho51 . 
Your thinking is correct. The scenario should reflect the business view or the reaility of the feature as seen by the user, and that in turn should drive the implementation. The error was just the mismatch between the implementation and the scenario. In my email response, I did say that the feature file had the wrong casing because of what the original case was before your change, but really, we should be using what's on the user interface (which is File Name).